boa/vyper/contract.py

```diff
@@ -1045,9 +1045,12 @@ def from_abi_dict(cls, abi, name=None):
 
         return cls(name, functions, events)
 
-    def at(self, address) -> ABIContract:
+    def at(self, address, name=None) -> ABIContract:
```

Comment:
i'm not sure this is necessary

Reply:
the idea was if you did ERC20.at("addr"), you could specify a name "USDC". otherwise it just says `ERC20 interface at 0xaddress". but that's okay I guess.